packages/cip2/src/types.ts

```diff
@@ -99,8 +99,6 @@ export type ProtocolParametersForInputSelection = Pick<
   'coinsPerUtxoWord' | 'maxTxSize' | 'maxValueSize' | 'minFeeCoefficient' | 'minFeeConstant'
 >;
 
-export type ProtocolParametersRequiredByInputSelection = Required<
```

Comment:
Just be careful of small stylistic changes in future, possibly as a result of your IDE, as any lint errors should be enforced in CI.